packages/blade/src/components/Checkbox/CheckboxGroup/useCheckboxGroup.ts

```diff
@@ -45,6 +45,7 @@ const useCheckboxGroup = ({
   const [checkedValues, setValue] = useControllableState({
     value,
     defaultValue: defaultValue || [],
+    // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
```

Comment:
Not sure why this started failing suddenly. I've ignored it instead of making it the following to avoid breaking changes and unintended behaviour changes

```diff
- onChange: (values: string[]) => onChange?.({ values, name: name! }),
+ onChange: (values: string[]) => onChange?.({ values, name: name }),
```